mycqu/card.py

```diff
@@ -149,25 +149,36 @@ class EnergyFees:
     """
     balance: float
     """账户余额"""
-    electricity_subsidy: float
-    """电剩余补助"""
-    water_subsidy: float
-    """水剩余补助"""
+    electricity_subsidy: Optional[float]
+    """电剩余补助（仅虎溪校区拥有）"""
+    water_subsidy: Optional[float]
+    """水剩余补助（仅虎溪校区拥有）"""
+    subsidies: Optional[float]
+    """补助余额（仅老校区拥有）"""
 
     @staticmethod
-    def from_dict(data: dict[str, Any]) -> EnergyFees:
+    def from_dict(data: dict[str, Any], isHuXi: bool) -> EnergyFees:
         """从反序列化的（一个）水电费 json 中获取水电费信息
 
         :param data: json 反序列化得到的字典
         :type data: dict[str, Any]
         :return: 学期信息对象
         :rtype: EnergyFees
         """
-        return EnergyFees(
-            balance=data["剩余金额"],
-            electricity_subsidy=data["电剩余补助"],
-            water_subsidy=data["水剩余补助"]
-        )
+        if isHuXi:
+            return EnergyFees(
+                balance=data["剩余金额"],
+                electricity_subsidy=data["电剩余补助"],
+                water_subsidy=data["水剩余补助"],
+                subsidies=None,
+            )
+        else:
+            return EnergyFees(
+                balance=data["现金余额"],
+                electricity_subsidy=None,
+                water_subsidy=None,
+                subsidies=data["补贴余额"],
+            )
 
     @staticmethod
     def fetch(session: Session, isHuxi: bool, room: str) -> EnergyFees:
```

Comment:
（这个应该是之前的问题，没注意到）
同上
```suggestion
    def fetch(session: Session, is_huxi: bool, room: str) -> EnergyFees:
```

此外文档字符串里请加入 `is_huxi` 的文档